Makefile

```diff
@@ -96,7 +98,8 @@ lint-fix: $(GOLANGCI_LINT) ## Lint the codebase and run auto-fixers if supported
 .PHONY: modules
 modules: ## Runs go mod to ensure modules are up to date.
 	go mod tidy
-	cd $(TOOLS_DIR); go mod tidy
+	cd $(TOOLS_DIR); go mod tidy; cd $(ROOT_DIR)
```

Comment:
```suggestion
	cd $(TOOLS_DIR); go mod tidy
```

Is that needed?

Reply:
Yes, unless i wrap `$(ENVTEST_DIR)` in `$(abspath)`, which i've just done.